packages/allure-playwright/src/index.ts

```diff
@@ -307,15 +310,12 @@ class AllureReporter implements Reporter {
     for (const group of this.allureGroupCache.values()) {
       group.endGroup();
     }
-    if (process.env.PW_ALLURE_POST_PROCESSOR_FOR_TEST) {
-      try {
-        const writer = this.allureWriter;
-        void writer; // Used in `eval()`, below.
-        const postProcess = eval(process.env.PW_ALLURE_POST_PROCESSOR_FOR_TEST); // eslint-disable-line no-eval
-        console.log(JSON.stringify(postProcess(this.allureWriter))); // eslint-disable-line no-console
-      } catch (e) {
-        console.log(JSON.stringify({ error: (e as Error).stack || String(e) })); // eslint-disable-line no-console
-      }
+
+    if (this.options.environmentInfo) {
+      this.allureRuntime?.writeEnvironmentInfo(this.options?.environmentInfo);
```

Comment:
why `onEnd`?

Reply:
All the misc files should be generated at the end of the run to allow changing them in the tests/fixtures. Currently, there is no runtime API for that, but I moved it just to align with the vision that such data should be delivered at the end